Flow.Launcher/MainWindow.xaml.cs

```diff
@@ -84,8 +85,33 @@ private void OnLoaded(object sender, RoutedEventArgs _)
                             QueryTextBox.SelectAll();
                             _viewModel.LastQuerySelected = true;
                         }
+
+                        if (_viewModel.ProgressBarVisibility == Visibility.Visible)
+                        {
+                            _progressBarStoryboard.Resume();
+                        }
                     }
+                    else
+                    {
+                        _progressBarStoryboard.Pause();
```

Comment:
This is going to be called everytime the mainwindow is hidden, how come a check is not added here?

Reply:
fixed in https://github.com/Flow-Launcher/Flow.Launcher/pull/295/commits/db63833e93c0b89290fa30a79533b82e28051dfc